Sources/LightweightCharts/Implementations/API/Series/BaselineSeries.swift

```diff
@@ -3,6 +3,6 @@ import Foundation
 public class BaselineSeries: SeriesObject, SeriesApi {
     
     public typealias Options = BaselineSeriesOptions
-    public typealias TickValue = SingleValueData
+    public typealias TickValue = BaselineData
```

Comment:
SingleValueData is not used anymore. Is it intentional?

https://github.com/search?q=repo%3Atradingview%2Flightweight-charts-android%20SingleValueData&type=code

Reply:
Also https://github.com/tradingview/lightweight-charts/blob/78cd426872119ea2b72cc0b3ad58a4fc87e67f13/src/api/data-consumer.ts#L53

Reply:
specific data for specific series. 

In Android use `SingleValueData` like interface.
In IOS protocol designed over `SingleValueSeriesData`.

Android no have simple implementation for the interface (`SingleValueData`)
IOS have `SingleValueData` as simple implementation `SingleValueSeriesData` 's protocol